mentorship ios/Members/Members.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Members.swift
+//  mentorship ios
+//
+//  Created by Yugantar Jain on 07/06/20.
+//  Copyright © 2020 Yugantar Jain. All rights reserved.
+//
+
+import SwiftUI
+
+struct Members: View {
+    @ObservedObject var membersModel = MembersModel()
+    var body: some View {
+        NavigationView {
+            List {
```

Comment:
screen padding here?

Reply:
The list view has an automatic padding. The list elements align with the navigation bar title.
I've tried adding additional padding, but it doesn't looks nice.
@annabauza